src/3_PB/AnalisiDeiRequisiti/content.typ

```diff
@@ -493,20 +531,17 @@ Relativamente all'utilizzo della #glossary("dashboard"), viene definito un unico
   caption: [UC5 Visualizzazione allerte superamento soglie]
 )
 
-/*TODO: andare a fare l'immagine apposta*/
-#pagebreak()
 === UC5.1: Visualizzazione allerta superamento soglia temperatura
 - *Attore Principale*: #glossary("amministratore pubblico")\;
 - *Precondizioni*: il #glossary("sistema") è operativo e accessibile;
 - *Postcondizioni*: l'#glossary("amministratore pubblico") riceve un'allerta relativa alla temperatura che notifica il superamento dei 40° Celsius (40°C);
 - *Scenario Principale*:
-  + Il #glossary("sistema") rileva una temperatura nella media dei 5 minuti, superiore ai 40° Celsius (40°C).
+  + Il #glossary("sistema") rileva una temperatura superiore ai 40° Celsius (40°C).
```

Comment:
non va specificato tipo che la temperatura deve stare alta per tot secondi?
e stessa cosa per gli altri

Reply:
non serve in realtà basta che vada su una volta e la notifica parte entro i 5 minuti successivi